### DISCUSSION TITLE
Progressive Enhancement e Graceful Degradation

### DISCUSSION THREAD

**Discussion by grodrigues** [frontendbr/forum#1370] [Perguntas e Respostas]
Boa Noite vocês utilizam os conceitos de Progressive Enhancement e Graceful Degradation nos projetos desenvolvidos ?

**Comment by felipefialho** [comment]
Uso sim mano, acho fundamental para construir aplicações usando técnicas novas e manter ele sem legado daqui algum tempo.

Tem alguma dúvida especifica?

**Comment by grodrigues** [comment]
Só essa dúvida mesmo, porque estava lendo algumas coisas sobre css Cross Broswer e me lembrei desses conceitos de extrema em importância no desenvolvimento e o conteúdo que encontrei na internet estava com o ano de 2014, 2013 e ninguém mais falou nada sobre isso por isso fique com essa dúvida.

**Comment by evertonthepaula** [comment]
Eu costumo avaliar quais browsers o usuário final vai utilizar, então deixo o trabalho duro para os pré-processadores, compiladores, transpiladores e geradores de bundle! :D

**Comment by gss-patricia** [comment]
Meu modus operandi é igual ao do @evertonthepaula :D

**Comment by felipefialho** [comment]
Essa é a melhor abordagem mesmo @evertonthepaula
